config/jest/setupTests.js

```diff
@@ -1,4 +1,4 @@
 import { configure } from 'enzyme'
-import Adapter from 'enzyme-adapter-react-16'
+import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
```

Comment:
Official `enzyme-adapter-react-17` is WIP.  See https://github.com/enzymejs/enzyme/issues/2429